docs/pages/setup/reference/cli.mdx

```diff
@@ -1310,6 +1310,346 @@ $ tctl auth rotate --type=user --grace-period=200h
+The test process is safe from side effects in that:
+- it will not change the list of configured auth connectors
+- the audit log will clearly state the login attempts as "test" ones
+- there will be no certificates issued for logged in user
+
+<Admonition
+  type="note"
+  title="Important"
+>
+  To use this command, you must have access to `github_request`, `oidc_request`, `saml_request` resources (depending on the type of connector being tested).
+
+  If you receive a "permission denied" error, ensure that you have access to the following resources in one of your Teleport roles:
+
+```yaml
+- resources: [github_request]
+  verbs: [list,create,read,update,delete]
+- resources: [oidc_request]
+  verbs: [list,create,read,update,delete]
+- resources: [saml_request]
+  verbs: [list,create,read,update,delete]
+```
+</Admonition>
+
+
+#### Arguments
+
+- `[<filename>]` Connector resource definition file. Optional. Empty for stdin.
+
+#### Flags
+
+This command defines no flags.
+
+#### Global flags
+
+These flags are available for all commands `--debug, --config`. Run
+`tctl help <subcommand>` or see the [Global Flags section](#tctl-global-flags).
+
+#### Examples
+
+Test the auth connector from `connector.yaml`:
+
+```code
+$ tctl sso test connector.yaml
+```
+
+The command is designed to be used in conjunction with the `tctl sso configure` family of commands:
+
+```code
+$ tctl sso configure ... | tctl sso test
+```
+
+The pipeline may also utilize `tee` to capture the connector generated with `tctl sso configure`. This will save the connector in `connector.yaml`:
+
+```code
+$ tctl sso configure ... | tee connector.yaml | tctl sso test
+```
+
+You can test an existing auth connector by combining the command with `tctl get`:
+
+```bsh
+$ tctl get saml/your-connector-name --with-secrets | tctl sso test
+````
+
+<Admonition
+  type="note"
+  title="Important"
+>
+  Make sure to include `--with-secrets` flag, or the exported auth connector will not be testable.
+</Admonition>
+
+### tctl sso configure github
+
+Configure GitHub auth connector.
+
+Required params `--id` and `--secret` come from the [Github OAuth app](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app).
+
+The flag `--teams-to-roles` can be provided multiple times to specify which Github Teams are assigned which roles.
+In this example, members of the `octocats` team in GitHub will be assigned the `devs`, `access`, and `editor` roles in Teleport.
+
+```code
+$ tctl sso test --id=GITHUB_CLIENT_ID --secret=GITHUB_SECRET --teams-to-roles=octocats,devs,access,editor [<other-flags>]
+```
+
+#### Arguments
+
+This command accepts no arguments.
+
+#### Flags
+
+Mandatory flags: `--id`, `--secret`, `--teams-to-roles`.
+
+| Name                      | Default Value(s) | Allowed Value(s)           | Description                                             |
```

Comment:
"Github" should be "GitHub" in this table.

Reply:
Replaced "Github" with "GitHub" here and elsewhere.